readme.md

```diff
@@ -1,60 +1,148 @@
 # Official Chia Docker Container
 
-## Basic Startup
-```
-docker run --name <container-name> -d ghcr.io/chia-network/chia:latest
-(optional -v /path/to/plots:/plots)
+## Quick Start
+
+These examples shows valid setups using Chia for both docker run and docker-compose. Note that you should read some documentation at some point, but this is a good place to start.
+
+### Docker run
+
+```bash
+docker run --name chia -d ghcr.io/chia-network/chia:latest --expose=8447 -v /path/to/plots:/plots
 ```
-#### set the timezone for the container (optional, defaults to UTC)
-Timezones can be configured using the `TZ` env variable. A list of supported time zones can be found [here](http://manpages.ubuntu.com/manpages/focal/man3/DateTime::TimeZone::Catalog.3pm.html)
+Syntax
+```bash
+docker run --name <container-name> -d ghcr.io/chia-network/chia:latest 
+optional accept incoming connections: --expose=8447
+optional: -v /path/to/plots:/plots
 ```
--e TZ="America/Chicago"
+
+### Docker compose
+
+```yaml
+version: "3.6"
+services:
+  chia:
+    container_name: chia
+    restart: always
```

Comment:
```suggestion
    restart: unless-stopped
```

IMO this is a better default, in case someone has intentionally stopped the container, it wont restart in that case

Reply:
Make sense